certified-connectors/IN-D Payables/apiDefinition.swagger.json

```diff
@@ -1,3 +1,808 @@
+
```

Comment:
@gowthamd3 Please remove this empty lines

Reply:
@sriyen-msft removed empty lines

Reply:
@gowthamd3 Empty lines are still present